src/components/SignerPanel/SigningStatus.js

```diff
@@ -38,14 +34,15 @@ class SigningStatus extends React.Component {
     status: SignerStatusType.isRequired,
     signError: PropTypes.instanceOf(Error),
     walletProviderId: PropTypes.string,
+    theme: PropTypes.object,
   }
   getLabel() {
     const { status } = this.props
     if (isSigning(status)) return 'Waiting for signature…'
     if (status === STATUS_TX_SIGNED) return 'Transaction signed!'
     if (status === STATUS_MSG_SIGNED) return 'Message signed!'
-    if (status === STATUS_TX_ERROR) return 'Error signing the transaction.'
-    if (status === STATUS_MSG_ERROR) return 'Error signing the message.'
+    if (status === STATUS_TX_ERROR) return 'Transaction failed!'
+    if (status === STATUS_MSG_ERROR) return 'Message failed!'
```

Comment:
cc @dizzypaty the copy for these are a bit misleading, because it's not the transaction or the message that have failed at this step.

Suggestions:

- `Signing transaction failed!`
- `Signing message failed!`